test/e2e/tests/01-apply-recommendations/00-assert.yaml

```
@@ -0,0 +1,35 @@
+status:
+  phase: Running
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: test-workloads
+  labels:
+    app: busybox
+status:
+  phase: Running
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)